### PR TITLE
Gender and date of birth to the user added.

### PR DESCRIPTION
**Pull request by pacificsun**
Hi, Ian here is the PR Please review it and any feedback will be appreciated! :)

### CODE REVIEW

File: integration-testing/tests/users/properties.test.js [hunk 1]
```diff
@@ -88,6 +110,12 @@ test('Disabled user cannot setUserDetails', async () => {
   await expect(client.mutate({mutation: mutations.setUserDetails, variables: {bio: 'a dog'}})).rejects.toThrow(
     /ClientError: User .* is not ACTIVE/,
   )
+  await expect(client.mutate({mutation: mutations.setUserDetails, variables: {dob: '1995-05-05'}})).rejects.toThrow(
+    /ClientError: User .* is not ACTIVE/,
+  )
+  await expect(client.mutate({mutation: mutations.setUserDetails, variables: {gender: 'Male'}})).rejects.toThrow(
+    /ClientError: User .* is not ACTIVE/,
+  )
```
Comment (mfogel) [inline integration-testing/tests/users/properties.test.js:118]: I wouldn't bother to test that `dob` and `gender` throw errors when using `setUserDetails()` here. The test is focused on testing if disabling users works, so I think it's sufficient to test that by trying to set one random property to something... that random property could be `bio` or `dob` or `gender`, doesn't really matter which. But all three is overkill, in my opinion.

File: real-main/schema.graphql [hunk 2]
```diff
@@ -32,7 +32,11 @@ type Query {
   user(userId: ID!): User
```
Comment (mfogel) [inline real-main/schema.graphql:32]: It looks like your auto-formatter did a lot to this file. The new format is not necessarily good or bad, but it doesn't belong in this PR. Better to separate re-formats of files into separate PR's.

File: real-main/schema.graphql [hunk 3]
```diff
@@ -98,18 +123,20 @@ type Mutation {
...
-    viewCountsHidden: Boolean,
-    commentsDisabled: Boolean,
-    likesDisabled: Boolean,
-    sharingDisabled: Boolean,
-    verificationHidden: Boolean,
-    languageCode: String,
+    username: String
+    fullName: String
+    bio: String
+    dob: String
```
Comment (mfogel) [inline real-main/schema.graphql:129]: Better to use AWS's custom scalar date type `AWSDate` here: https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html

File: real-main/schema.graphql [hunk 4]
```diff
@@ -98,18 +123,20 @@ type Mutation {
...
+    gender: String
```
Comment (mfogel) [inline real-main/schema.graphql:130]: Better to type this a as new custom graphql enum. Possible values would be at least `M` and `F`, and any others we want to add.

**Comment by pacificsun** [comment]
Thank you for the feedback and i will try to improve those points :)
